core/core_test.go

```diff
@@ -57,8 +57,8 @@ func TestForwardToLeader(t *testing.T) {
 		httpClient: &http.Client{
 			Transport: &http.Transport{
 				TLSClientConfig: &tls.Config{
-					RootCAs: certpool,
-				},
+					RootCAs:    certpool,
```

Comment:
Is this necessary for the tests to pass? Would we expect all clients to set this value in order to interact with Chain Core?